packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -6,6 +6,16 @@ import 'package:meta/meta.dart';
 
 import 'print.dart';
 
+/// Returns a 5 character long hexadecimal string generated from
+/// Object.hashCode's 20 least-significant bits.
+String shortHash(Object object) {
+  return object.hashCode.toUnsigned(20).toRadixString(16).padLeft(5, '0');
+}
+
+/// Returns a summary of the runtime type and hash code of `object`.
```

Comment:
similarly maybe this should link to shortHash, and maybe instead of "runtime type" and "hash code" it should hyperlink to the actual members on Object.